scalafix-testkit/src/main/scala/scalafix/testkit/SyntacticRuleSuite.scala

```diff
@@ -8,16 +8,20 @@ import scalafix.syntax._
 
 import org.scalatest.FunSuiteLike
 
-class SyntacticRuleSuite(rule: Rule) extends FunSuiteLike with DiffAssertions {
-  def check(name: String, original: String, expected: String): Unit = {
```

Comment:
Forward this to new `check` overload with `defaultRule`

Reply:
Done.